core/src/com/unciv/ui/pickerscreens/FoundReligionPickerScreen.kt

```diff
@@ -0,0 +1,103 @@
+package com.unciv.ui.pickerscreens
+
+import com.badlogic.gdx.scenes.scene2d.ui.*
+import com.badlogic.gdx.utils.Align
+import com.unciv.UncivGame
+import com.unciv.logic.GameInfo
+import com.unciv.logic.civilization.CivilizationInfo
+import com.unciv.models.UncivSound
+import com.unciv.models.ruleset.Belief
+import com.unciv.models.translations.tr
+import com.unciv.ui.utils.*
+import kotlin.math.max
+
+class FoundReligionPickerScreen (
+    private val choosingCiv: CivilizationInfo,
+    private val gameInfo: GameInfo
+): PickerScreen() {
+
+    // Roughly follows the layout of the original (although I suck at UI designing, so please improve this)
+    private val topReligionIcons = Table() // Top of the layout, contains icons for religions
+    private val leftChosenBeliefs = Table() // Left middle part, contains buttons to select the types of beliefs to choose
+    private val rightBeliefsToChoose: ScrollPane // Right middle part, contains the beliefs to choose
+    
+    private val middlePanes: SplitPane
+    private val newPanes: SplitPane
+ 
+    private var previouslySelectedIcon: Button? = null
+    private var iconName: String? = null
+    private var religionName: String? = null
+    private var chosenFounderBelief: Belief? = null
+    private var chosenFollowerBelief: Belief? = null
+
+    init {
+        closeButton.isVisible = true
+        setDefaultCloseAction()
+        
+        stage.clear()
```

Comment:
No need for this

Reply:
Or maybe yes? I don't understand what you're trying to do here
If you want a plain screen, use CameraStageBaseScreen

Reply:
> I don't understand what you're trying to do here

Neither do I :)
I wanted to reuse the bottom of the picker screen, as it gives some basic functionallity. However, I was unable to get anything working when I just added elements to the topTable, so in the end I decided to take the PickerScreen, remove it entirely, and create a new splitpane with my own thing at the top, and the BottomBar of the pickerscreen at the bottom.